rust/example-threads/src/main.rs

```diff
@@ -0,0 +1,51 @@
+use std::{panic, thread};
+use std::fmt::Error;
+use std::thread::JoinHandle;
+use std::time::Duration;
+
+fn example_thread() {
+    println!("Start main thread");
+
+    // start a thread
+    let thread = thread::spawn(|| {
+        println!("Start other thread");
```

Comment:
Idee für etwas formatting sugar: 
```suggestion
        println!("  Start other thread");
```